js/templates/userPage/store.html

```diff
@@ -27,10 +27,10 @@ <h3><%= ob.polyT('userPage.store.unableToFetchListings') %></h3>
   <div class="flex">
     <div class="col storeFilters <% if (!ob.listingCount) print('disabled') %>">
       <div class="clrP clrBr padMd clrT clrSh2 contentBox">
-        <div class="txB row"><%= ob.polyT('userPage.store.shippingFilter.heading') %></div>
-        <div class="flexVCent tx5 rowSm">
+        <div class="txB rowSm"><%= ob.polyT('userPage.store.shippingFilter.heading') %></div>
+        <div class="flexVCent rowSm">
           <label class="margRSm" for="shipsToSelect"><%= ob.polyT('userPage.store.shippingFilter.shipsTo') %>:</label>
-          <select class="tx6 select2Small js-shipsToSelect" style="width: 133px">
+          <select class="tx6 select2Small js-shipsToSelect" style="width: 133px" id="shipsToSelect">
```

Comment:
Is the id necessary? It was added in, but I'm not seeing anything added in that uses it.

Reply:
The label uses it for the "for" attribute. Previously the for wasn't pointing to anything.